src/main/java/org/cryptomator/common/mount/MountWithinParentUtil.java

```diff
@@ -68,16 +72,47 @@ static void prepareParentNoMountPoint(Path mountPoint) throws MountPointPreparat
 		}
 	}
 
+	private static boolean removeResidualJunction(Path path) throws MountPointPreparationException {
+		try {
+			if (!Files.exists(path, LinkOption.NOFOLLOW_LINKS)) {
+				return false;
+			}
+			if (!SystemUtils.IS_OS_WINDOWS) { //So far this is only a problem on Windows
+				return true;
+			}
```

Comment:
remove those checks and just catch and handle NoSuchFileException thrown by  `Files.readAttributes(...)`.

Reply:
Resolved by 96ebb67085c17960772bdaddb70faf58c81f9c90.